webapp/app/assessment/fra/sections/areaAffectedByFire/areaAffectedByFireValidatorState.js

```diff
@@ -0,0 +1,32 @@
+import * as FRA from '@common/assessment/fra'
+
+import * as AssessmentStateValidator from '@webapp/app/assessment/assessmentStateValidator'
+
+const section = FRA.sections['5'].children.b
+
+export const totalForestLandAreaAreaValidator = AssessmentStateValidator.subCategoryValidator(
+  FRA.type,
```

Comment:
maybe we should use named params here
ex.
{ type: FRA.type,
name: section.name,
....
??: 0
??: [1]
}